### PR TITLE
Pmi 75 sybase integration test

### PR DESCRIPTION
**Pull request by redcatbear**
Summary:
* Switched Logging from System.out to `java.util.logging`
* Created customer formatter `com.exasol.logging.CompactFormatter`
* Local logging configurable via standard `logging.properties`
* Remote logging configurable in `CREATE VIRTUAL SCHEMA` statement
* Refactored and fixed output stream connection to socket
* Updated and cleaned-up documentation
* One document per dialect
* Improved test coverage
* Split SQL scripts for Sybase into phases for faster testing cycles

### CODE REVIEW

File: jdbc-adapter/README.md [hunk 1]
```diff
@@ -1,41 +1,34 @@
...
+* How to **map the tables** in the source systems to virtual tables in Exasol, including how to **map the data types** to Exasol data types.
+* How is the **SQL syntax** of the data source, including identifier quoting, case-sensitivity, function names, or special syntax like `LIMIT` / `TOP`.
+* Which **capabilities** are supported by the data source. E.g. is it supported to run filters, to specify select list expressions, to run aggregation or scalar functions or to order or limit the result.
 
-If you are interested in a introduction to virtual schemas please refer to the EXASOL user manual. You can find it in the [download area of the EXASOL user portal](https://www.exasol.com/portal/display/DOWNLOAD/6.0).
+In addition to the aforementioned dialects there is the so called `GENERIC` dialect, which is designed to work with any JDBC driver. It derives the SQL dialect from the JDBC driver metadata. However, it does not support any capabilities and might fail if the data source has special syntax or data types, so it should only be used for evaluation purposes.
```
Comment (andrehacker) [inline jdbc-adapter/README.md]: I like the sentence, but it makes more sense if we directly mention the dialects above

File: jdbc-adapter/README.md [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
 
+If you are interested in a introduction to virtual schemas please refer to the Exasol user manual. You can find it in the [download area of the Exasol user portal](https://www.exasol.com/portal/display/DOWNLOAD/6.0).
```
Comment (andrehacker) [inline jdbc-adapter/README.md]: Improvement proposal: Here is a better link: https://www.exasol.com/portal/display/DOC/Database+User+Manual. You could make a link out of `Exasol user manual`. (not really part of this PR - nice to have)

File: jdbc-adapter/doc/supported_sql_dialects.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Supported Dialects
```
Comment (andrehacker) [inline jdbc-adapter/doc/supported_sql_dialects.md]: See my first comment about this new subpage

File: jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/IntegrationTestSetup.java [hunk 4]
```diff
@@ -15,65 +17,142 @@
...
+ * </p>
  *
- * Attention: This does not deploy the latest jar, because the dist maven module is cleaned and build after this module. Right now you need to do something like "mvn clean package && mvn verify -Pit -D..." to upload and test the latest jar.
+ * <p>
+ * <strong>Attention:</strong>
+ * </p>
+ *
+ * <p>
+ * This does not deploy the latest JAR, because the distribution Maven module is
+ * and and build after this module. Right now you need to do something like
```
Comment (andrehacker) [inline jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/IntegrationTestSetup.java]: `and and` typo.

Right now I wonder if it is not possible to do it all in one Maven call. Maybe the integration tests can be moved into the pom.xml on highest level and this will be executed at the end? Not sure, ... Anyway, such change is not part of this PR.

File: jdbc-adapter/README.md [hunk 5]
```diff
@@ -1,41 +1,34 @@
...
-* Impala
-* Oracle
-* Teradata
-* Redshift
-* DB2
-* SQL Server
-* PostgreSQL
+The JDBC adapter for virtual schemas allows you to connect to JDBC data sources like Hive, Oracle, Teradata, Exasol or any other data source supporting JDBC. It uses the well proven ```IMPORT FROM JDBC``` Exasol statement behind the scenes to obtain the requested data, when running a query on a virtual table. The JDBC adapter also serves as the reference adapter for the Exasol virtual schema framework.
+
+Check the [SQL dialect list](doc/supported_sql_dialects.md) to learn which SQL dialects the JDBC adapter currently supports
```
[the remark this answers is not in the document]
Comment (redcatbear) [reply jdbc-adapter/README.md]: Done.

File: jdbc-adapter/README.md (hunk 1 above)
Comment (redcatbear) [reply jdbc-adapter/README.md]: Done.

File: jdbc-adapter/README.md (hunk 2 above)
Comment (redcatbear) [reply jdbc-adapter/README.md]: Exchanged link.

File: jdbc-adapter/doc/supported_sql_dialects.md (hunk 3 above)
Comment (redcatbear) [reply jdbc-adapter/doc/supported_sql_dialects.md]: Done.

File: jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/IntegrationTestSetup.java (hunk 4 above)
Comment (redcatbear) [reply jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/IntegrationTestSetup.java]: The integration tests need to be redone from the ground on.
They need to automatically create a clean environment for each test, we need cleaner and most importantly more compact configuration and uniform testing across all dialects. While we are at it we also need to eliminate code duplication.

That is a separate PR for sure.

**Comment by redcatbear** [comment]
@andrehacker: Implemented your requested changes. Please check again.

**Comment by redcatbear** [comment]
Accidentally hit the "close" button. Reopened.
